batcher.go

```diff
@@ -392,9 +450,16 @@ func (r *Batcher) Start() (err error) {
 			case <-auditTimer.C:
 				// ensure that if the buffer is empty and everything should have been flushed, that target is set to 0
 				if len(r.buffer) < 1 && time.Since(lastFlushWithRecords) > r.maxOperationTime {
-					if r.trySetTargetToZero() {
-						r.emit(AuditFailEvent, 0, "an audit revealed that the target should be zero but was not.", nil)
-					} else {
+					targetIsZero := r.confirmTargetIsZero()
+					inflightIsZero := r.confirmInflightIsZero()
+					switch {
+					case !targetIsZero && !inflightIsZero:
+						r.emit(AuditFailEvent, 0, AuditMsgFailureOnTargetAndInflight, nil)
```

Comment:
This feels strange to emit an event that is a combination of two other events. I feel like this should just raise both of the other events instead adding a third "combo" event.

Reply:
I originally had it that way; I changed it because it seemed weird that a single audit could have 1 event (success), sometimes 1 event (fail), and sometimes multiple events (fail).